config/src/main/java/com/typesafe/config/impl/SimpleConfigObject.java

```diff
@@ -364,11 +364,15 @@ public AbstractConfigValue modifyChild(String key, AbstractConfigValue v) {
         });
     }
 
+
     @Override
     protected void render(StringBuilder sb, int indent, boolean atRoot, ConfigRenderOptions options) {
         if (isEmpty()) {
             sb.append("{}");
         } else {
+            String[] keys = keySet().toArray(new String[size()]);
+            Arrays.sort(keys, new com.typesafe.config.impl.AlphanumericComparator());
+
```

Comment:
if there's no reason to move this block of code around in the file, why don't we keep it where it was so the diff more accurately shows the change that was made?